libs/sections/src/lib/header/vl-header.section.ts

```diff
@@ -30,63 +27,110 @@ export class VlHeader extends LitElement {
-            loginUrl: { type: String, attribute: 'data-vl-login-url', reflect: true },
-            loginRedirectUrl: { type: String, attribute: 'data-vl-login-redirect-url', reflect: true },
-            logoutUrl: { type: String, attribute: 'data-vl-logout-url', reflect: true },
-            switchCapacityUrl: { type: String, attribute: 'data-vl-switch-capacity-url', reflect: true },
-            authenticatedUserUrl: { type: String, attribute: 'data-vl-authenticated-user-url', reflect: true },
+            identifier: {
+                type: String,
+                attribute: 'data-vl-identifier',
+                reflect: true,
+            },
+            development: {
+                type: Boolean,
+                attribute: 'data-vl-development',
+                reflect: true,
+            },
+            loginUrl: {
+                type: String,
+                attribute: 'data-vl-login-url',
+                reflect: true,
+            },
+            loginRedirectUrl: {
+                type: String,
+                attribute: 'data-vl-login-redirect-url',
+                reflect: true,
+            },
+            logoutUrl: {
+                type: String,
+                attribute: 'data-vl-logout-url',
+                reflect: true,
+            },
+            switchCapacityUrl: {
+                type: String,
+                attribute: 'data-vl-switch-capacity-url',
+                reflect: true,
+            },
+            authenticatedUserUrl: {
+                type: String,
+                attribute: 'data-vl-authenticated-user-url',
+                reflect: true,
+            },
         };
     }
 
     constructor() {
         super();
+
         this.loginUrl = '/sso/aanmelden';
         this.loginRedirectUrl = '/';
         this.logoutUrl = '/sso/afgemeld';
         this.switchCapacityUrl = '/sso/wissel_organisatie';
         this.authenticatedUserUrl = '/sso/ingelogde_gebruiker';
     }
 
-    injectHeader() {
-        const vlBody = document.querySelector('[is="vl-body"]');
-        (vlBody || document.body).insertAdjacentHTML('afterbegin', '<div id="header"></div>');
+    private get headerContainer() {
+        return document.querySelector('#header__container');
     }
 
-    vlwHeader() {
-        return document.querySelector('div[class=vlw__header]');
+    private injectHeaderContainer() {
+        const vlBody = document.querySelector('[is="vl-body"]');
+        (vlBody || document.body).insertAdjacentHTML(
+            'afterbegin',
+            '<div id="header__container"><div id="header"></div></div>'
+        );
     }
 
-    header() {
-        return document.querySelector('#header');
+    private observeWidgetIsAdded() {
+        const isHeader = (node: Node) => {
+            return (
+                (node as HTMLElement).tagName === 'HEADER' || (node.childNodes && [...node.childNodes].some(isHeader))
```

Comment:
als ik dit goed begrijp kan de afnemer hier ook met `<header>` tag werken?

Reply:
besproken; dit is gedaan op de manier die onze voorgangers ook gedaan hebben
widget van Digitaal Vlaanderen heeft `<header>` tag